src/components/ui/checkbox.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+
+import * as React from 'react';
+import * as CheckboxPrimitive from '@radix-ui/react-checkbox';
+import { Check } from 'lucide-react';
+
+import { cn } from '@/lib/utils';
+
+const Checkbox = React.forwardRef<
+  React.ElementRef<typeof CheckboxPrimitive.Root>,
+  React.ComponentPropsWithoutRef<typeof CheckboxPrimitive.Root>
+>(({ className, ...props }, ref) => (
+  <CheckboxPrimitive.Root
+    ref={ref}
+    className={cn(
+      'peer h-6 w-6 m-2 shrink-0 rounded border border-slate-400 ring-offset-background focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50 data-[state=checked]:bg-secondary data-[state=checked]:text-white data-[state=checked]:border data-[state=checked]:border-slate-400',
```

Comment:
According to Figma, the `border-width` is `1.5px`. Should we use `1px` because Tailwind CSS does not natively have `1.5px`?
<img width="1010" alt="image" src="https://github.com/Tascurator/tascurator-frontend/assets/124953279/5e6442e2-374d-4768-b2ac-aa08b7b15b4b">

Reply:
border-width set 2px at UI kit so I changed it to 2px.